dcrdtest/rpc_harness.go

```diff
@@ -214,6 +216,15 @@ func New(t *testing.T, activeNet *chaincfg.Params, handlers *rpcclient.Notificat
 	return h, nil
 }
 
+// ToggleKeepNodeDir flags the Harness on whether to keep or remove its node
+// dir after TearDown is called.
+//
+// This is NOT safe for concurrent access and MUST be called from the same
+// goroutine that calls SetUp and TearDown.
+func (h *Harness) ToggleKeepNodeDir(keep bool) {
```

Comment:
Not a big deal, but toggle to me implies it will toggle it, as in flip the current state.  Perhaps just `SetKeepNodeDir(keep bool)`.